entry-nosystemd.sh

```diff
@@ -46,6 +51,8 @@ function init_non_systemd()
 	fi
 }
 
+remove_buildtime_env_var
```

Comment:
This needs to persist otherwise the child processes won't have the correct uname